src/status_im/contact/core.cljs

```diff
@@ -4,7 +4,7 @@
    [status-im.multiaccounts.model :as multiaccounts.model]
    [status-im.transport.filters.core :as transport.filters]
    [status-im.contact.db :as contact.db]
-   [status-im.contact.device-info :as device-info]
+
```

Comment:
extra line?

Reply:
I naively though that cljfmt would remove them